planner/core/physical_plans.go

```diff
@@ -329,6 +330,7 @@ func (p *PhysicalIndexReader) Clone() (PhysicalPlan, error) {
 		return nil, err
 	}
 	cloned.OutputColumns = util.CloneCols(p.OutputColumns)
+	cloned.CommonHandleCols = util.CloneCols(p.CommonHandleCols)
```

Comment:
Should `PhysicalIndexLookUpReader.Clone()` be similar?

Reply:
this logic is removed